.github/workflows/tests.yml

```diff
@@ -31,6 +31,8 @@ jobs:
           access_token: ${{ github.token }}
 
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2

```

Comment:
Why are more than the latest commit fetched here? Any particular reason for this change?

Reply:
I noticed that this was the reason codecov was failing 😆 It's why, as of today, we're no longer waiting for the codecov status to come in

Reply:
Oh, do you know the reason for that?